roundware/rw/templates/rw/model_text_display_div.html

```diff
@@ -0,0 +1 @@
+</div>
```

Comment:
I'm all for reducing code duplication, but does this single `</div>` need to be in a separate file?